operator-metadata/manifests/bundle.clusterserviceversion.yaml

```diff
@@ -376,6 +376,8 @@ metadata:
     features.operators.openshift.io/token-auth-aws: "false"
     features.operators.openshift.io/token-auth-azure: "false"
     features.operators.openshift.io/token-auth-gcp: "false"
+    operators.openshift.io/infrastructure-features: |-
```

Comment:
This is not need, it should be covered by the annotations above it

Reply:
It shouldn't cause any issues though

Reply:
@kyguy I get an error when I remove it as it's covering `["disconnected", "proxy", "proxy-aware", "fips"]`

Reply:
Removed